crates/pink-libs/subrpc/src/lib.rs

```diff
@@ -375,7 +367,19 @@ mod tests {
         let signer: [u8; 32] =
             hex!("9eb2ee60393aeeec31709e256d448c9e40fa64233abf12318f63726e9c417b69");
         let remark = "Greetings from unit tests!".to_string();
-        let signed_tx = create_transaction(&signer, "khala", rpc_node, 0u8, 1u8, remark);
+        let signed_tx = create_transaction(
+            &signer,
+            "khala",
+            rpc_node,
+            0u8,
+            1u8,
+            remark,
+            ExtraParam {
+                era: None,
```

Comment:
Nice to turn it into ExtraParam::default() if possible.

Reply:
Done